hangupsbot/commands.py

```diff
@@ -483,9 +484,17 @@ def stock(bot, event, *args):
             tickers)
         # Cant use get_json because of 3 invalid chars
         data = json.loads(raw_data.text[3:])
-        result = '\n'.join([i['t'] + ": " + i['l'] for i in data])
+        for i in data:
+            ticker_link = hangups.ChatMessageSegment(
+                i['t'],
+                hangups.SegmentType.LINK,
+                link_target="https://www.google.com/finance?q=" + i['t']
+            )
+            result.append(ticker_link)
+            result.append(': ' + i['l'] + ' (' + i['cp'] + '%)')
```

Comment:
Shouldn't this be:

    result.append(
        hangups.ChatMessageSegment(': ' + i['l'] + ' (' + i['cp'] + '%)')
    )